frontend/src/pages/instance/Assets.vue

```diff
@@ -97,15 +112,11 @@ export default {
         }
     },
     mounted () {
-        if (!this.hasAMinimumTeamRoleOf('member')) {
-            return this.$router.push({ name: 'instance-overview' })
-        }
-
         this.loadContents()
     },
     methods: {
         loadContents () {
-            if (this.isFeatureEnabled) {
+            if (this.isFeatureEnabled && this.hasAMinimumTeamRoleOf('member')) {
```

Comment:
Can we use the enum instead of hardcoded string?

e.g. `RoleNames[Roles.member]`

Better still if `hasAMinimumTeamRoleOf` accepted the actual role enum value e.g. `hasAMinimumTeamRoleOf(Roles.member)`

This way, any refactoring will be caught and references can be found.